internal/runners/prepare/prepare.go

```diff
@@ -37,29 +34,18 @@ func New(prime primeable) *Prepare {
 func (r *Prepare) Run() error {
 	logging.Debug("ExecutePrepare")
 
-	binDir := filepath.Join(config.CachePath(), "bin")
-	fail := fileutils.Mkdir(binDir)
-	if fail != nil {
-		return locale.WrapError(fail.ToError(), "err_prepare_bin_dir", "Could not create bin directory")
-	}
-
-	envUpdates := map[string]string{
-		"PATH": binDir,
-	}
-
-	fail = r.subshell.WriteUserEnv(envUpdates, sscommon.Default, true)
-	if fail != nil {
+	if err := globaldefault.Prepare(r.subshell); err != nil {
 		if runtime.GOOS != "linux" {
-			return locale.WrapError(fail.ToError(), "err_prepare_update_env", "Could not update user environment")
+			return locale.WrapError(err, "err_prepare_update_env", "Could not prepare environment.")
```

Comment:
The error key `err_prepare_update_env` has already been used inside of the `globaldefault.Prepare` function...